Pod/Classes/VOKCoreDataManager.h

```diff
@@ -22,6 +22,12 @@ typedef NS_ENUM (NSInteger, VOKMigrationFailureOption) {
     VOKMigrationFailureOptionWipeRecoveryAndAlert,
 };
 
+/// The action block for asynchronous writing to a temporary context.
+typedef void(^VOKWriteBlock)(NSManagedObjectContext *tempContext);
+
+/// A completion block after an asynchronous operation on a temporary context. NSManagedObjectID's are threadsafe.
+typedef void(^VOKObjectIDReturnBlock)(NSArray *arrayOfManagedObjectIDs);
```

Comment:
Should it be like "attorneys general"? 
`VOKObjectsIDReturnBlock` or `VOKObjectIDsReturnBlock`?